src/components/Map/USACountyMap.js

```diff
@@ -10,9 +10,26 @@ import { useSummaries } from 'common/location_summaries';
 import { ScreenshotReady } from 'components/Screenshot';
 
 function getStateCode(stateName) {
+  // hacky, name defined slightly differently in normal reversed states file.
+  if (stateName === 'Commonwealth of the Northern Mariana Islands') {
+    return 'MP';
+  }
   return REVERSED_STATES[stateName];
 }
 
+const MarianaIslands = ({ x, y, fill, scale, onMouseEnter, onMouseLeave }) => {
+  return (
+    <g
+      transform={`translate(${x}, ${y}) scale(${scale})`}
+      onMouseEnter={onMouseEnter}
+      onMouseLeave={onMouseLeave}
+    >
+      <rect width="20" height="20" fill={fill} />
+      <text transform="translate(25, 15)">CNMI</text>
```

Comment:
Is this label just temporary?

Reply:
The box with CNMI is the UX that we settled on for now.  It is possible that we may change it in the future, but no immediate plans

Reply:
Added a comment above